src/import-data/ImportData.test.ts

```diff
@@ -16,10 +16,9 @@ import { selectExistingWorkspacePrompt } from './ImportDataDestination';
 
 type UserEvent = ReturnType<typeof userEvent.setup>;
 
-type WorkspaceUtilsExports = typeof import('src/workspaces/common/state/useWorkspaces');
-jest.mock('src/workspaces/common/state/useWorkspaces', (): WorkspaceUtilsExports => {
+type useWorkspacesExports = typeof import('src/workspaces/common/state/useWorkspaces');
+jest.mock('src/workspaces/common/state/useWorkspaces', (): useWorkspacesExports => {
   return {
-    ...jest.requireActual<WorkspaceUtilsExports>('src/workspaces/common/state/useWorkspaces'),
```

Comment:
@nawatts @msilva-broad is the preference to _always_ include the "requireActual" even if the thing being mocked only has a single export (which is being mocked by the test)?

Reply:
I would vote yes. It's a good habit to get into since it works for all modules: you don't have to think about how many exports a module has and make a decision based on that when writing a test. Also, it's future proof against a module later adding more exports.